src/ModalPopupComponent.coffee

```diff
@@ -2,56 +2,118 @@ React = require 'react'
 ReactDOM = require 'react-dom'
 H = React.DOM
 _ = require 'lodash'
+Modal = require 'react-overlays/lib/Modal'
+className = require "react-overlays/node_modules/classnames"
```

Comment:
Reaching inside node_modules is a big no-no (and doesn't work with my npm). Add `classnames` to package.json and use it normally instead.